dev-python/pythondialog/pythondialog-3.5.1.ebuild

```diff
@@ -0,0 +1,39 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{6,7,8} pypy3 )
+
```

Comment:
`2.2.2` uses `rdepend` for `DISTUTILS_SETUP_TOOLS`, maybe it still is needed?

Did you get a warning when trying to build a package?